kubernetes-deployment/maintainlib/common.py

```diff
@@ -79,12 +79,12 @@ def write_generated_file(generated_file, file_path):
 
 def sftp_paramiko(src, dst, filename, host_config):
 
-    hostip = host_config['hostip']
-    username = host_config['username']
-    password = host_config['password']
+    hostip = str(host_config['hostip'])
```

Comment:
>hostip [](start = 4, length = 6)

please check the format of hostip is valid

Reply:
Another feature, do it in the feature in our plan.